modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -14,8 +19,24 @@ import * as ViewBlock from '../module/test/ViewBlock';
 describe('browser.tinymce.core.EditorManagerTest', () => {
   const viewBlock = ViewBlock.bddSetup();
 
+  const assertIsUuid = (uuid?: string): void => {
+    // From https://github.com/uuidjs/uuid/blob/main/src/regex.js
+    const v4Regex = /^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i;
+    assert.isString(uuid);
+    // Need to narrow type
+    if (!Type.isString(uuid)) {
+      assert.fail();
+    }
+    assert.lengthOf(uuid, 36);
+    assert.match(uuid, v4Regex);
+    const version = parseInt(uuid.slice(14, 15), 16);
+    assert.equal(version, 4);
+  };
```

Comment:
Please add further validation, specifically
https://datatracker.ietf.org/doc/html/rfc4122#section-4.4

> Set the two most significant bits (bits 6 and 7) of the clock_seq_hi_and_reserved to zero and one, respectively.

The way we've done this in the past is by verifying with the bytes, not the string, using `const bits = bytes[8].toString(2).padStart(8, "0")` and then verifying `bits.substring(0, 2)` is `"10"`. So it might be easier to have that as a specific uuid test rather than trying to parse it out of the string afterwards 🤔

Reply:
I'll see what I can do 👍